example-doc-win2019.md

```diff
@@ -0,0 +1,276 @@
+# Windows Server 2019
+
+## Installed Software
+### Languages and Runtimes
+- Java 1.7: build 1.7.0_262-b10
+- Java 1.8: build 1.8.0_252-b14 (default)
+- Java 11: build 11.0.7+10-LTS
+- Rust 1.43.0
+- Python 3.7.7
+- Ruby 2.5.8p224
+- Go 1.14.2
+- PHP 7.4.5
+- julia 1.4.1
+- Perl 5.30.2
+- PowerShell Core 7.0.0
+- Node.js v12.16.3
+
+### Package Management
+- Chocolatey 0.10.15
+- Vcpkg 2020.02.04
+- NPM 6.14.4
+- Yarn 1.22.4
+- Pip 20.0.2 (python 3.7.7)
+- Miniconda 4.6.14
+- RubyGems 3.1.2
+- Helm v3.2.0+ge11b7ce
+- Composer 1.10.5 2020-04-10 11:44:22
+
+### Project Management
+- Ant 1.10.5
+- Maven 3.6.3
+- Gradle 6.3
+
+### Tools
+- Azure CosmosDb Emulator 2.9.2.0
+- Bazel 3.1.0
+- Bazelisk 1.4.0
+- CMake 3.17.1
+- Docker 19.03.5
+- Docker-compose 1.25.4
+- Git: 2.26.2
+- Git LFS: 2.10.0
+- Inno Setup 6.0.4
+- jq 1.6
+- Kubectl v1.18.2
+- Kind v0.7.0
+- MinGW 8.1.0
+- MySQL 5.7.21.0
+- Mercurial 5.0
+- NSIS 3.5
+- Packer 1.5.5
+- SQLServer PS 21.1.18221
+- SQLPS 1.0
+- Subversion (SVN) 1.8.17
+- VSWhere 2.8.4
+- WinAppDriver 1.1.1809.18001
+- zstd 1.4.0
+
+### CLI Tools
+- Azure CLI 2.5.0
+- Azure DevOps CLI extension 0.18.0
+- AWS SAM CLI 0.48.0
+- AWS CLI 2.0.10
+- Alibaba Cloud CLI 3.0.26
+- Cloud Foundry CLI 6.51.0
+- Hub CLI: 2.14.2
+
+### Browsers and webdrivers
+- Google Chrome 81.0.4044.122
+- Chrome Driver 81.0.4044.69
+- Microsoft Edge 81.0.416.64
+- Microsoft Edge Driver 81.0.416.64
+- Mozilla Firefox 75.0
+- Gecko Driver 0.26.0
+- IE Driver 3.8.0.0
+
+### Toolcache (architecture x64)
```

Comment:
Maybe rename it to Languages toolcache and move under the ### Languages and Runtimes section? @maxim-lobanov @AlenaSviridenko thoughts?